Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCapturer.cpp

```diff
@@ -385,10 +385,10 @@ void GStreamerVideoCapturer::reconfigure()
                 selector->maxWidth = *width;
                 selector->maxHeight = *height;
                 selector->maxFrameRate = *frameRate;
-                selector->mimeType = gstStructureGetName(structure).toString();
+                selector->mimeType = String(gstStructureGetName(structure).span());
                 if (gst_structure_has_name(structure, "video/x-raw")) {
                     if (gst_structure_has_field(structure, "format"))
-                        selector->format = gstStructureGetString(structure, "format"_s).toString();
+                        selector->format = String(gstStructureGetString(structure, "format"_s).span());
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?